components/hidden-wallet-summary.js

```diff
@@ -1,22 +1,20 @@
-import { useState, useRef, useEffect } from 'react'
+import { useState, useRef } from 'react'
 import { abbrNum, numWithUnits } from '@/lib/format'
 import { useMe } from './me'
 
 export default function HiddenWalletSummary ({ abbreviate, fixedWidth }) {
   const me = useMe()
   const [hover, setHover] = useState(false)
 
-  // prevent layout shifts when hovering by fixing width to initial rendered width
   const ref = useRef()
-  const [width, setWidth] = useState(undefined)
-  useEffect(() => {
-    setWidth(ref.current?.offsetWidth)
-  }, [])
+  // prevent layout shifts when hovering by fixing width to initial rendered width of '******'
+  // We can simply multiply 6 by '0.6em' since monopsace has a fixed width for all characters
+  const width = '3.6em'
 
   return (
     <span
-      ref={ref} style={{ width: fixedWidth ? width : undefined }}
-      className='text-monospace' align='right' onPointerEnter={() => setHover(true)} onPointerLeave={() => setHover(false)}
+      ref={ref} style={{ display: 'block', width: fixedWidth ? width : undefined, textAlign: 'right' }}
```

Comment:
Ahh, didn't catch that this should have been `display: 'inline-block'`

Reply:
I tried that too but then it doesn't work in the navbar! It's a tricky mistress.

Reply:
hmm couldn't break it like that did but yours works just the same :+1: